controllers/topics-controller.js

```diff
@@ -0,0 +1,9 @@
+const { fetchTopics } = require("../models/topics-model");
+
+exports.getTopics = (req, res, next) => {
+  fetchTopics()
+    .then((topics) => {
+      res.status(200).send({ topics });
+    })
+    .catch(next);
```

Comment:
You shouldn't need to catch errors as the user is not providing any parameters or queries.